docs/_source/tutorials_and_integrations/tutorials/feedback/monitoring-bias-ethics-rlhf.ipynb

```diff
@@ -0,0 +1,628 @@
+    "\n",
+    "\n",
+    "**Launch Argilla using Argilla's quickstart Docker image**: This is the recommended option if you want [Argilla running on your local machine](../../getting_started/quickstart.html). Note that this option will only let you run the tutorial locally and not with an external notebook service.\n",
+    "\n",
+    "For more information on deployment options, please check the Deployment section of the documentation.\n",
+    "\n",
+    "<div class=\"alert alert-info\">\n",
+    "\n",
+    "Tip\n",
+    "    \n",
+    "This tutorial is a Jupyter Notebook. There are two options to run it:\n",
+    "\n",
+    "- Use the Open in Colab button at the top of this page. This option allows you to run the notebook directly on Google Colab. Don't forget to change the runtime type to GPU for faster model training and inference.\n",
+    "- Download the .ipynb file by clicking on the View source link at the top of the page. This option allows you to download the notebook and run it on your local machine or on a Jupyter Notebook tool of your choice.\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Set up the Environment\n",
+    "\n",
+    "To complete this tutorial, you will need to install the Argilla client and a few third-party libraries using `pip`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# %pip install --upgrade pip\n",
+    "%pip install argilla -qqq\n",
+    "%pip install \"giskard[llm]\" --upgrade\n",
+    "%pip install \"langchain<=0.0.301\" \"pypdf<=3.17.0\" \"faiss-cpu<=1.7.4\" \"openai<=0.28.1\" \"tiktoken<=0.5.1\"\n",
+    "%pip install avidtools"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's make the needed imports:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import argilla as rg\n",
+    "from argilla.feedback import TrainingTask\n",
+    "from argilla.feedback import ArgillaTrainer\n",
+    "\n",
+    "import os\n",
+    "import json\n",
+    "import pandas as pd\n",
+    "import openai\n",
+    "from pathlib import Path\n",
+    "from typing import Any, Dict, Iterator, Tuple\n",
+    "\n",
+    "from langchain.llms import OpenAI\n",
+    "from langchain.chains.base import Chain\n",
+    "from langchain.vectorstores import FAISS\n",
+    "from langchain.prompts import PromptTemplate\n",
+    "from langchain.embeddings import OpenAIEmbeddings\n",
+    "from langchain.document_loaders import PyPDFLoader\n",
+    "from langchain.chains import RetrievalQA, load_chain\n",
+    "from langchain.text_splitter import RecursiveCharacterTextSplitter\n",
+    "\n",
+    "from giskard import Dataset, Model, scan"
```

Comment:
By convention, inside each of the blocks, we follow alphabetical order for imports. Doesn't the formatted automatically, does it? If not, is just moving around some stuff :) I can push myself the change if you are unsure.